voter/management/commands/voter_fetch.py

```diff
@@ -1,139 +1,60 @@
-            os.makedirs(target_folder, exist_ok=True)
-            write_success = write_stream(resp, target_filename)
-            if write_success:
-                status_code = FETCH_STATUS_CODES.CODE_OK
-            else:
-                status_code = FETCH_STATUS_CODES.CODE_WRITE_FAILURE
-        else:
-            status_code = FETCH_STATUS_CODES.CODE_NET_FAILURE
-    return (status_code, etag, now, target_filename)
+    E.g.
 
+    no arg: fetch latest statewide voter and voter history snapshot.
 
-def process_new_zip(url, base_path, label, county_num=None):
-    print("Fetching {0}".format(label))
-    fetch_status_code, etag, created_time, target_filename = attempt_fetch_and_write_new_zip(url, base_path)
-    if fetch_status_code == FETCH_STATUS_CODES.CODE_OK:
-        print("Fetched {0} successfully to {1}".format(label, target_filename))
-        print("Extracting {0}".format(target_filename))
-        unzip_success = extract_and_remove_file(target_filename)
-        if unzip_success:
-            target_dir = os.path.dirname(target_filename)
-            for filename in os.listdir(target_dir):
-                if filename.endswith(".txt"):
-                    result_filename = os.path.join(target_dir, filename)
-                    print("Finished extracting to {0}".format(result_filename))
-            print("Updating FileTracker table")
-            if label == 'ncvoter':
-                data_file_kind = FileTracker.DATA_FILE_KIND_NCVOTER
-            else:
-                data_file_kind = FileTracker.DATA_FILE_KIND_NCVHIS
-            FileTracker.objects.create(
-                etag=etag, filename=result_filename,
-                county_num=county_num, created=created_time,
-                data_file_kind=data_file_kind)
-        else:
-            print("Unable to unzip {0}".format(target_filename))
-            return FETCH_STATUS_CODES.CODE_WRITE_FAILURE
-    else:
-        if fetch_status_code == FETCH_STATUS_CODES.CODE_NET_FAILURE:
-            print("Unable to fetch file from {0}".format(url))
-        if fetch_status_code == FETCH_STATUS_CODES.CODE_WRITE_FAILURE:
-            print("Unable to write file to {0}".format(target_filename))
-        if fetch_status_code == FETCH_STATUS_CODES.CODE_NOTHING_TO_DO:
-            print("Resource at {0} contains no new information. Nothing to do.".format(url))
-    return fetch_status_code
-
-
-class Command(BaseCommand):
-    help = "Fetches and processes voter and voter history data from NCSBE.gov"
+    --bycounty: fetch county files rather than statewide
+    """
 
     def add_arguments(self, parser):
+        # Don't rewrap the text in the help/description:
+        parser.formatter_class = argparse.RawDescriptionHelpFormatter
+
         parser.add_argument(
             '--bycounty',
             action='store_true',
             dest='bycounty',
             default=False,
             help='Fetch per county files rather than statewide',)
+        parser.add_argument(
```

Comment:
Should this get added to the help docstring above where `--bycounty` was documented?